### ISSUE TITLE
Please update containerd.io to version 1.2.3 (random container crash due to containerd.io bug)

### ISSUE THREAD

**Issue by mman** [docker/for-linux#586]
Docker containers that have ``HEALTHCHECK`` configured can crash randomly due to race described at https://github.com/containerd/containerd/issues/2969.

Fix is available in https://github.com/containerd/containerd/releases/tag/v1.2.3.

* [x] This is a bug report
* [ ] This is a feature request
* [x] I searched existing issues before opening this one

### Expected behavior

Docker containers with ``HEALTHCHECK`` running indefinitely.

### Actual behavior

Docker containers with ``HEALTHCHECK`` crashing randomly due to a bug in containerd.io.

**Output of `docker version`:**

```
$ docker --version
Docker version 18.09.1, build 4c52b90
```

**Output of `docker info`:**

```
$ docker info
Containers: 3
 Running: 3
 Paused: 0
 Stopped: 0
Images: 76
Server Version: 18.09.1
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 9754871865f7fe2f4e74d43e2fc7ccd237edcbce
runc version: 96ec2177ae841256168fcf76954f7177af9446eb
init version: fec3683
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.15.0-45-generic
Operating System: Ubuntu 18.04.2 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 985.5MiB
Name: bmx-staging
ID: 7KB6:BXYY:36W6:HC67:ERS5:XMZ5:NTEU:T4FZ:NHWZ:2TWK:E2JU:XQXD
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine

WARNING: No swap limit support
```

**Comment by seemethere** [comment]
We are expecting to do a release of this very soon, will check back in when it's live.

**Comment by thaJeztah** [comment]
containerd 1.2.5 is available now (and 1.2.6 should be coming in the not too distant future);

```
apt-cache madison containerd.io
containerd.io |    1.2.5-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io |    1.2.4-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io |    1.2.2-3 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io |    1.2.2-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io |    1.2.0-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io | 1.2.0~rc.2-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io | 1.2.0~rc.0-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
containerd.io | 1.2.0~beta.2-1 | https://download.docker.com/linux/ubuntu bionic/edge amd64 Packages
```
